### ISSUE TITLE
Questions About Datasets

### ISSUE THREAD

**Issue by YYX666660** [yxlu-0102/AP-BWE#10]
:)
After reviewing the training and inference code, I would like to ask if the validation set and the test set are the same? 
The paper doesn't seem to mention this part.

**Comment by yxlu-0102** [comment]
I referred to some open-sourced codes, they seem to use the same test set as the validation set, but it's not recommended, you can just randomly select 500 utterances from the training set as the validation set.

**Comment by YYX666660** [comment]
Got it! thx :)
